go.mod

```diff
@@ -923,3 +923,11 @@ retract v0.37.0 // Contains dependencies on v0.36.0 components, which should hav
 
 // see https://github.com/distribution/distribution/issues/3590
 exclude github.com/docker/distribution v2.8.0+incompatible
+
+// Replacing prometheus/client_golang so that cmd/otelcontribcol/main_testbed.go passes. This is needed because the test runs exporter/prometheusexporter that requires the exemplars feature in github.com/prometheus/client_golang.
+// This replace can be removed once the exemplars feature (https://github.com/prometheus/client_golang/pull/986) is available in a release version and this go.mod is updated to use that.
+replace github.com/prometheus/client_golang => github.com/prometheus/client_golang v1.12.2-0.20220318110013-3bc8f2c651ff
```

Comment:
Why is this added? I believe this was reverted in https://github.com/open-telemetry/opentelemetry-collector-contrib/commit/282dbe70239e057f6f9246eb1e60e3bca9e8c828#diff-cbc1780ee44619ffeeb078f84991774b14445d3a192c34b6acc709f00788b421 and few tests passed after that

Reply:
No idea, must have missed it in the rebase.

Reply:
Rebased again to grab latest, and made sure this didn't get excluded. Got a little funky with all the conflicts/reverts.